tests/basic-request.json

```diff
@@ -1,28 +1,28 @@
 {
-    "past_articles": [
-        {
-            "article_id": "e7605f12-a37a-4326-bf3c-3f9b72d0738d",
-            "title": "title 1",
-            "content": "content 1",
-            "url": "url 1"
-        }
-    ],
-    "todays_articles": [
-        {
-            "article_id": "7e5e0f12-d563-4a60-b90a-1737839389ff",
-            "title": "title 2",
-            "content": "content 2",
-            "url": "url 2"
-        }
-    ],
-    "interest_profile": {
-        "click_history": {
-            "account_id": "977a3c88-937a-46fb-bbfe-94dc5dcb68c8",
-            "article_ids": [
-                "e7605f12-a37a-4326-bf3c-3f9b72d0738d"
-            ]
-        },
-        "onboarding_topics": []
+  "past_articles": [
+    {
+      "article_id": "e7605f12-a37a-4326-bf3c-3f9b72d0738d",
+      "title": "title 1",
+      "content": "content 1",
+      "url": "url 1"
+    }
+  ],
+  "todays_articles": [
+    {
+      "article_id": "7e5e0f12-d563-4a60-b90a-1737839389ff",
+      "title": "title 2",
+      "content": "content 2",
+      "url": "url 2"
+    }
+  ],
+  "interest_profile": {
+    "profile_id": "9eb70e32-fff9-4e8a-a05b-df582174fbf5",
```

Comment:
We use `profile_id` instead of `account_id` here because the id supplied by the platform side may not actually be the account id if we're running an experiment recommender since we aren't allowed to track accounts across experiments.